regional_mom6/regional_mom6.py

```diff
@@ -886,39 +896,54 @@ def bathymetry(
                 make topography (if true), or read an existing file.
             positivedown (Optional[bool]): If true, assumes that
                 bathymetry vertical coordinate is positive down.
+            chunks (Optional Dict[str, str]): Chunking scheme for bathymetry, eg {"lon": 100, "lat": 100}. Use lat/lon rather than the coordinate names in the input file.
 
 
         """
 
         if maketopog == True:
-            bathy = xr.open_dataset(
-                bathy_path, chunks={varnames["xh"]: 500, varnames["yh"]: 500}
-            )[varnames["elevation"]]
+            if chunks != "auto":
+                chunks = {varnames["xh"]: chunks["lon"], varnames["yh"]: chunks["lat"]}
+
+            bathy = xr.open_dataset(bathy_path, chunks=chunks)[varnames["elevation"]]
 
             bathy = bathy.sel(
-                {varnames["yh"]: slice(self.yextent[0] - 0.1, self.yextent[1] + 0.1)}
+                {varnames["yh"]: slice(self.yextent[0] - 1, self.yextent[1] + 1)}
```

Comment:
Why do we need to remove either 0.1 or 1 from here?

Reply:
OK, I see.. you just take a bit more of the bathymetry than the domain.
I think we should either:
- have this buffer size be automatically adjusted based on the resolution the user provides + the resolution of the bathymetry
_or_
- at least have a comment here so that we know that there is a hard-coded assumption that 1 degree buffer is enough
-

Reply:
Yeah lets do the latter for now 
```suggestion
                {varnames["yh"]: slice(self.yextent[0] - 1, self.yextent[1] + 1)} #! Hardcoded 1 degree buffer around bathymetry selection. Should automatically select buffer in future
```